kubernetes/jobs/padigitale-report-generator/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/name: {{SERVICE_NAME}}
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/interop-iam-service-{{LOWERCASE_ENV}}-{{SERVICE_NAME}}
```

Comment:
in case you already haven't, remember that this role needs to be created in infra repository

Reply:
@micdes-pagopa may I ask for you help here?